ADR/user-specified-tokens-build.md

```diff
@@ -0,0 +1,28 @@
+# Enable users to configure tokens/credentials for a Build/Test Pipeline
+
+Date: 2023-03-03
+
+## Status
+
+Provisional
+
+
+## Context
+
+As a user, I would like to configure sensitive information/tokens/credentials for the build/test pipeline to consume as part of a partner task.
+
+
+## Decision
+
+1. UI will take in sensitive information as form input and upload it to SPI using the user's authentication token. See https://github.com/redhat-appstudio/service-provider-integration-operator/blob/main/docs/USER.md#uploading-access-token-to-spi-using-kubernetes-secret .
+2. In the above form, the user should be able to choose the name of the `Secret` from a dropdown. 
+The dropdown would be a pre-populated list of `Secret` names associated with known partner integrations. The user should also be allowed to provide a name barring the ones that already exist in the namespace.
```

Comment:
How this list of Secrets will be provided?

Reply:
This would be a hardcoded list of parameter names.